CurrencyConverter/Java-SpringBoot/src/main/resources/currency.xsd

```diff
@@ -0,0 +1,21 @@
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:tns="http://campuswien.edu/se/carrental/currencyconverter"
+           targetNamespace="http://campuswien.edu/se/carrental/currencyconverter" elementFormDefault="qualified">
+
+    <xs:element name="convertRequest">
+        <xs:complexType>
+            <xs:sequence>
+                <xs:element name="dollar" type="xs:double"/>
+                <xs:element name="currency" type="xs:string"/>
```

Comment:
Kannst du bitte Currency mit Enum schreiben?
wir haben ein Beispiel in spring sample.

Reply:
ja mache ich